Lucidtech/Las/Client.cs

```diff
@@ -11,6 +11,16 @@
 
 namespace Lucidtech.Las
 {
+    class OptionalParameter {
+        private string? Name { get; set; }
```

Comment:
Does it make sense that the name is empty? Doesn't that always need to be a string?